man/systemd.link.xml

```diff
@@ -356,6 +356,16 @@
           </para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>AlternativeNamesPolicy=</varname></term>
+        <listitem>
+          <para>An ordered, space-separated list of policies by which the interface's alternative names
```

Comment:
"ordered"? The order shouldn't really matter much, no? the policies listed here are independent of order, any order is fine, and effectively equivalent, no?

Reply:
This is a copy and paste mistake. Dropped.